### DISCUSSION TITLE
How do I hide navbar-brand on main page

### DISCUSSION THREAD

**Discussion by antforager** [bootscore/bootscore#908] [Q&A] [answered]
Hi, 

I'm on v6.03 of Bootscore, and 6.0.0-beta1 of Bootscore Child.

I want to hide the `navbar-brand` logo just on the main page, which has a different top part of the page. So, the "AntGear" logo on the top-left of https://antgear.com should be hidden. If you browse to https://antgear.com/product-category/ants/, for example, it should display the `navbar-brand` logo. 

Note: I'm using the free Widget Context plugin but this is not a widget. It's part of the header. Though it seems like an appropriate use case for that plugin. Moving along...

In Header.php, we have this:

```
...
<!-- Navbar Brand -->
        <a class="navbar-brand" href="<?= esc_url(home_url()); ?>">
          <img src="<?= esc_url(apply_filters('bootscore/logo', get_stylesheet_directory_uri() . '/assets/img/logo/logo.svg', 'default')); ?>" alt="<?php bloginfo('name'); ?> Logo" class="d-td-none me-2">
          <img src="<?= esc_url(apply_filters('bootscore/logo', get_stylesheet_directory_uri() . '/assets/img/logo/logo-theme-dark.svg', 'theme-dark')); ?>" alt="<?php bloginfo('name'); ?> Logo" class="d-tl-none me-2">
        </a> 
...
```

That part is in the header, although I see in source code that it used to be in `/template-parts/header/navbar-brand.php`, but it's not there anymore. I don't want to override the entire `header.php`.

Is there a way to make a filter or something to do this? I just need to check if the location is home_url and if it is, exclude writing the logo code.

Thank you in advance.

Cindy

**Comment by crftwrk** [comment, marked as answer]
Why so complicated? Simply add this into the page content. Go in Backend to your page edit screen, add direct after the title a Custom HTML Block and paste code to hide logo on this page there:


```html
<style>
.navbar-brand {
  display: none;
}
</style>
```

<img width="404" alt="Bildschirmfoto 2024-12-23 um 12 58 52" src="https://github.com/user-attachments/assets/4c802b59-9add-4c1d-bffd-f1d7c6a65dc0" />

Now, the logo is hidden only on this page. Solved?

**Comment by antforager** [comment]
Thank you.

**Reply by crftwrk** [reply]
You're welcome 👍 Let's keep this open, other users may have the same question. Please mark the answer, thank you.
